exp-player/addon/components/jam-auth/template.hbs

```diff
@@ -0,0 +1,20 @@
+<div class="well">
+    <p>Enter your <strong>Study ID</strong> and <strong>Participant ID</strong> into the fields below. Please {{link-to 'contact us' 'contact'}} if you have any questions.</p>
```

Comment:
I would prefer to see the ISP-specific representation of this component included in that repo instead. 

Can you please replace this with a more generic version? The ISP version can create a subclass of this component and override `layout` to have this template.